ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/platform.py

```diff
@@ -51,15 +51,23 @@
             "mlag": 300,
             "non_mlag": 330,
         },
-        "feature_support": {"queue_monitor_length_notify": False, "poe": True},
+        "feature_support": {"queue_monitor_length_notify": False, "poe": True, "per_interface_mtu": False},
     },
     {
         "platforms": ["720DP", "722XP", "710P"],
```

Comment:
Last I checked some platforms in this group was using a different chip. It is unfortunately not the whole family we should enforce this on. This is why we have not addressed the issue. We needed someone to map out all the affected platforms.